src/activities/menu/ConfigurationItem.qml

```diff
@@ -404,6 +413,58 @@ Item {
                 }
             }
         }
+        
+        Flow {
```

Comment:
Not sure if we need this row in the configuration. We don't  have it for the wordset. Maybe we handle it silently and under the hood. Not sure about it, let's keep it for now unless Johnny has an opinion.